salt/cloud/clouds/msazure.py

```diff
@@ -660,9 +674,9 @@ def wait_for_hostname():
         ret['Attached Volumes'] = created
 
     data = show_instance(vm_['name'], call='action')
-    log.info('Created Cloud VM \'{0[name]}\''.format(vm_))
+    log.info('Created Cloud VM {0[name]!r}'.format(vm_))
```

Comment:
The repr flag ``!r`` here is not acceptable, please revert all changes which use it.